client/src/Login.css

```diff
@@ -0,0 +1,111 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;700&display=swap');
+
+.photo {
+    width: 270px
+}
+
+
+.Logo {
+    display: flex;
+    justify-content: center;
+    padding-top: 3cm;
+}
+
+.inputTitle {
+    text-align: left;
+    font-family: 'Montserrat', sans-serif;
+    color: #81A1B7;
+    font-size: 14pt;
+    font-weight: 700;
+    margin: 5px;
+}
+
+.orText {
+    display: flex;
+    justify-content: center;
+    color: rgb(0, 132, 156);
+    font-weight: 700;
+    font-family: 'Montserrat', sans-serif;
+}
+
+.forgetPass {
+    display: flex;
+    justify-content: center;
+    font-family: 'Montserrat', sans-serif;
+    font-weight: 400;
+}
+
+.forgetPass a:link, a:visited {
+    color: rgb(0, 132, 156);
+    font-weight: 400;
+    padding-top: 20px;
+    font-size: 16pt;
```

Comment:
A bit weird that the "forgot password" text is bigger than the rest of the text on the page. Let's make it a bit smaller. And let's put `font-size`, along with the `margin-top` (as discussed above), on line 36 together with the `font-family` and `font-weight`.